appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -620,6 +620,14 @@
   @Description("Label of the link for About")
   String aboutMenuItem();
 
+  @DefaultMessage("Use Companion")
+  @Description("Label showing the companion version")
+  String useCompanion();
+
+  @DefaultMessage("Target Android SDK")
+  @Description("Label showing the target sdk version and the target android version")
+  String targetSdkVersion();
```

Comment:
One improvement that could be made here is to use parameterized messages. For example, below you concatenate in colon and parentheses, but you could do this instead:

```java
@DefaultMessage("Target Android SDK: {0} ({1})")
@Description("Label showing the target sdk version and the target android version")
String targetSdkVersion(String sdk, String name);
```

When you need the string in TopToolbar, you would just pass the values in as parameters.

This will allow different languages to potentially reorder the values if necessary. You can also apply this technique to the Use Companion string above as well.